packages/tailwindcss/src/utilities.ts

```diff
@@ -628,7 +634,10 @@ export function createUtilities(theme: Theme) {
   staticUtility('col-span-full', [['grid-column', '1 / -1']])
   functionalUtility('col-span', {
     themeKeys: [],
-    handleBareValue: ({ value }) => value,
+    handleBareValue: ({ value }) => {
```

Comment:
I put the validation for `col-span`, `col-start`, `col-end`, `row-span`, `row-start`,  and `row-end` in a separate commit because they can technically almost contain any value. If a "name" is used, then they would refer to named grid areas: https://developer.mozilla.org/en-US/docs/Web/CSS/grid-column-start
```css
/* <custom-ident> value */
grid-column-start: somegridarea;

/* <integer> + <custom-ident> values */
grid-column-start: 2;
grid-column-start: somegridarea 4;
```

So the question is, do we want to allow anything here, or is it fine to only accept integer values for bare values, e.g.: `col-span-123` and if you want to generate values using names that you should use arbitrary values instead?

Reply:
I kinda think just do numbers and use arbitrary values for other stuff 👍🏻

Reply:
Agree, I think bare values should just be numbers. Feels much simpler to explain that too.

Reply:
Alright perfect, than I can keep the commit in there, nice 👍